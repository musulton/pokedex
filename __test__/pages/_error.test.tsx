import {render} from "@testing-library/react";

import Error from "../../src/pages/_error";

describe("Error page", () => {
    it("should render 404 error correctly", () => {
        const {container} = render(<Error statusCode={404} />)

        expect(container.firstChild).toMatchSnapshot();
    })

    it("should render internal server error correctly", () => {
        const {container} = render(<Error />)

        expect(container.firstChild).toMatchSnapshot();
    })
})
