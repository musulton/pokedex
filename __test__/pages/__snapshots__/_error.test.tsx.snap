// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error page should render 404 error correctly 1`] = `
<div
  class="sc-249c5c88-0 bHOtmB"
>
  <p
    class="sc-249c5c88-1 dUHQTv"
  >
    404 | Resource not found...
  </p>
</div>
`;

exports[`Error page should render internal server error correctly 1`] = `
<div
  class="sc-249c5c88-0 bHOtmB"
>
  <p
    class="sc-249c5c88-1 dUHQTv"
  >
    Internal server error...
  </p>
</div>
`;
